de.blobbyvolley.BlobbyVolley2.desktop

```diff
@@ -0,0 +1,16 @@
+[Desktop Entry]
```

Comment:
Should be upstreamed

Reply:
https://github.com/danielknobe/blobbyvolley2/issues/16